account_payment_return_import/wizard/payment_return_import_view.xml

```diff
@@ -6,7 +6,13 @@
         <field name="model">payment.return.import</field>
         <field name="priority">1</field>
         <field name="arch" type="xml">
-            <form string="Import Payment Returns">
+            <form>
+                <div name="info_alert" class="alert alert-info" role="alert">
+                    How to import your payment return :
```

Comment:
Is this going to be translatable this way? I think you have to enclose it in a tag (`<span>` for example). Previous method was OK. Why change it?

Reply:
@pedrobaeza Odoo 12 does not accept `<label>` without `for` attribute.

`<label string= "1. Download your Payment Return file from your bank website."/>` doesn't work (error while validating the view).
`<label string= "1. Download your Payment Return file from your bank website." for="date_file"/>` would work just fine but is confusing. 

I'll check the translation of my way to do.

Reply:
Right, but you can put the same `for` for all. The problem is that changing this, you will waste all the done translations.

Reply:
@pedrobaeza Well, a migration is the best opportunity to get rid of an ugly code/interface. We'll only lost the translation of 4 sentences. Moreover, it will be far easier to translate them by putting all 4 sentences in a div instead of separate labels.

Reply:
OK, go ahead if the translation works.